aardvark/view.py

```diff
@@ -22,6 +24,31 @@ def __init__(self):
         super(RoleSearch, self).__init__()
         self.reqparse = reqparse.RequestParser()
 
+    def combine(self, aa):
+        del aa['count']
+        del aa['page']
+        del aa['total']
+
+        usage = dict()
+        for arn, services in aa.items():
+            for service in services:
+                namespace = service.get('serviceNamespace')
+                last_authenticated = service.get('lastAuthenticated')
+                if namespace not in usage:
+                    usage[namespace] = service
+                else:
+                    if last_authenticated > usage[namespace]['lastAuthenticated']:
+                        usage[namespace] = service
+                        del usage[namespace]['totalAuthenticatedEntities']
```

Comment:
`totalAuthenticatedEntities` doesn't seem to be useful in a combination scenario unless we were to calculate the SUM